modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance.jsp

```diff
@@ -30,6 +30,7 @@ FormInstancePermissionCheckerHelper formInstancePermissionCheckerHelper = ddmFor
 
 		<liferay-ui:search-container
 			id="<%= ddmFormAdminDisplayContext.getSearchContainerId() %>"
+			rowChecker="<%= new EmptyOnClickRowChecker(renderResponse) %>"
```

Comment:
@carolmariaabb, why this line was necessary?

Reply:
@natocesarrego After deleting multiple Forms by using the Checkbox next to 'Filter and Order', a message like "X of 0 Selected" (_X > 0_) would appear in the management bar. 
So I included this line to fix this bug.

Reply:
Good job, @carolmariaabb! Thanks for your answer.